service/s3/internal/customizations/updateEndpoint.go

```diff
@@ -0,0 +1,156 @@
+package customizations
```

Comment:
nit file naming pattern between implementation and test is inconsistent. would be good if they followed the same pattern, e.g. `update_endpoint.go`, `update_endpoint_test.go`